hq/app/controllers/SnykController.scala

```diff
@@ -0,0 +1,75 @@
+package controllers
+
+
+import auth.SecurityHQAuthActions
+import play.api.Configuration
+import play.api.libs.ws.WSClient
+import play.api.mvc._
+
+import scala.concurrent.ExecutionContext
+import logic.SnykDisplay
+import api.Snyk
+import com.amazonaws.services.simplesystemsmanagement.AWSSimpleSystemsManagement
+import com.gu.configraun.Errors.ConfigraunError
+import utils.attempt.{Attempt, FailedAttempt, Failure}
+import utils.attempt.PlayIntegration.attempt
+import model._
+
+class SnykController(val config: Configuration)
+                    (implicit
+                     val ec: ExecutionContext,
+                     val wsClient: WSClient,
+                     val bodyParser: BodyParser[AnyContent],
+                     val controllerComponents: ControllerComponents,
+                     val assetsFinder: AssetsFinder,
+                     val awsClient: AWSSimpleSystemsManagement,
```

Comment:
Why are these two implicit?

Reply:
The awsClient shouldn't be there at all.  The configraun client is ... arguable, I suppose.

It could be a 'proper' variable, but I imagined it would be sent everywhere and had some idea that would be a good reason for making it implicit.  I've made it 'proper' instead.